### PR TITLE
Complete homework #2

### PR DESCRIPTION
**Pull request by AnastasiiaNashchokina**

### CODE REVIEW

**Comment by AntBelyi** [comment]
What is that? Why do we need it?
Screenshot_6

File: src/main/java/HW_2/home_task_from_metanit/App.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package HW_2.home_task_from_metanit;
```
Comment (AntBelyi) [inline src/main/java/HW_2/home_task_from_metanit/App.java]: Please remove this extra internal folder
Screenshot_7

File: src/main/java/HW_2/home_task_from_metanit/App.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        int cc = aa - bb; // 6
+        // *
+        int s = 15;
+        int k = 2;
+        int f = s * k; // 30
+        // /
+        double p=40/2.5; //16.0
+        // remainder of two numbers
+        int q = 32 % 5; // 2 (32 - 5 * 6 = 2)
+        // prefix increment (спочатку збільшує значення зиінної на 1, а потім присвоює значення)
```
Comment (AntBelyi) [inline src/main/java/HW_2/home_task_from_metanit/App.java]: There should be no Latin words

File: src/main/java/HW_2/home_task_from_metanit/App.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        int m = 3;
+        int n = ++m;
+        System.out.println(m); // 4
+        System.out.println(n); // 4
+        //postfix increment (спочатку присвоює значення, а потім збільшує значення змінної на 1)
```
Comment (AntBelyi) [inline src/main/java/HW_2/home_task_from_metanit/App.java]: There should be no Latin words

File: src/main/java/HW_2/home_task_from_metanit/App.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        int e = 5;
+        int h = e++;
+        System.out.println(e); // 6
+        System.out.println(h); // 5
+        /* prefix decrement (спочатку зменшує значення змінної на 1, а потім присвоює значення)
+        postfix decrement (спочатку присвоює значення, а потім зменшує значення змінної на 1
```
Comment (AntBelyi) [inline src/main/java/HW_2/home_task_from_metanit/App.java]: There should be no Latin words

File: src/main/java/HW_2/home_task_from_metanit/App.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        boolean d2 = a2 != 10; // false
+
+        int a3 = 15;
+        int b3 = 9;
+        boolean c3 = a3 < b3; // false
+
+        int a4 = 20;
+        int b4 = 4;
+        boolean c4 = a4 > b4; // true
+    // >= true, якщо перший операнд більше або дорівнює другому, інакше - false
```
Comment (AntBelyi) [inline src/main/java/HW_2/home_task_from_metanit/App.java]: There should be no Latin words

**Comment by AntBelyi** [comment]
THis task isn't completed
Screenshot_8

File: src/main/java/HW_2/home_task_from_metanit/App.java (hunk 2 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_2/home_task_from_metanit/App.java]: fixed

File: src/main/java/HW_2/home_task_from_metanit/App.java (hunk 3 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_2/home_task_from_metanit/App.java]: fixed

File: src/main/java/HW_2/home_task_from_metanit/App.java (hunk 4 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_2/home_task_from_metanit/App.java]: fixed

File: src/main/java/HW_2/home_task_from_metanit/App.java (hunk 5 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_2/home_task_from_metanit/App.java]: fixed

File: src/main/java/HW_2/home_task_from_metanit/App.java (hunk 1 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_2/home_task_from_metanit/App.java]: fixed

**Comment by AnastasiiaNashchokina** [comment]
> THis task isn't completed Screenshot_8

done in " Task2"

File: src/main/java/HW_2/task2.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+public class task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+
+        System.out.println(x + y);
+        System.out.print(x);
+        System.out.println(y);
+    }
+}
```
Comment (AntBelyi) [inline src/main/java/HW_2/task2.java:10]: Implementatiuon is incorrect 
image


Yo need only one System.out.println that displays result in one line

Comment (AnastasiiaNashchokina) [reply src/main/java/HW_2/task2.java:10]: fixed

**Comment by AnastasiiaNashchokina** [comment]
> What is that? Why do we need it? Screenshot_6

fixed

**Pull request merged**